swarm/storage/ldbstore.go

```diff
@@ -143,13 +158,10 @@ func NewLDBStore(params *LDBStoreParams) (s *LDBStore, err error) {
 	}
 	data, _ := s.db.Get(keyEntryCnt)
 	s.entryCnt = BytesToU64(data)
-	s.entryCnt++
```

Comment:
I remember that other code was depending on these counters, for some reason, to be incremented when initialized. Did you make sure that we are not messing up with the counters now?

Reply:
This is coming from @nonsense 's fix, which is not merged yet to master: https://github.com/ethersphere/go-ethereum/pull/896